ssz-rs/src/merkleization/mod.rs

```diff
@@ -2,14 +2,12 @@ mod cache;
 mod node;
 mod proofs;
 
-use crate::ser::{Serialize, SerializeError};
+use crate::lib::*;
+use crate::ser::Serialize;
 use lazy_static::lazy_static;
 use sha2::{Digest, Sha256};
-use std::cmp::Ordering;
-use std::fmt::Debug;
-use std::ops::Index;
-use thiserror::Error;
 
+use crate::SerializeError;
```

Comment:
why grab this from the top-level, instead of leaving the import as it was?

Reply:
Reverted.